blockauth/jwt/token_manager.py

```
@@ -73,53 +73,26 @@ def generate_token(
                 provider_claims = provider.get_custom_claims(user)
-                logger.info(f"✅ Provider {provider.__class__.__name__} returned claims: {provider_claims}")
 
                 if provider_claims:
                     custom_claims.update(provider_claims)
-                    logger.info(
-                        f"✅ Added custom claims from {provider.__class__.__name__}: {list(provider_claims.keys())}"
-                    )
-                else:
-                    logger.info(f"⚠️ Provider {provider.__class__.__name__} returned no claims")
+            except user_model.DoesNotExist:
+                logger.warning("User %s not found, skipping claims provider %s", user_id, provider.__class__.__name__)
```

Ruff (0.15.9)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)